src/utils/compose-classes.js

```diff
@@ -0,0 +1 @@
+export default (...args: any) => args.filter(Boolean).join(' ')
```

Comment:
just realized, why do we use sometimes camel case and sometimes dash separated file names? In JSS I stick to camel case.